packages/karma/plugins.js

```diff
@@ -0,0 +1,9 @@
+module.exports = [
```

Comment:
could use a comment explaining why we pick these plugins, and how this file ensures we get the right version

Reply:
done.